test/cpp/jit/test_gpu.cpp

```diff
@@ -2602,7 +2602,7 @@ void testGPU_FusionSimpleBCast() {
     tv4->axis(0)->parallelize(ParallelType::BIDx);
     tv4->axis(-1)->parallelize(ParallelType::TIDx);
 
-    size_t x = 63, y = 33, z = 15;
+    constexpr int x = 63, y = 33, z = 15;
```

Comment:
MSCV complains about size_t here:

..\test\cpp\jit\test_gpu.cpp(2609): error C2398: Element '1': conversion from 'size_t' to '_Ty' requires a narrowing conversion
        with
        [
            _Ty=int64_t
        ]